### ISSUE TITLE
Create translation tables/example scripts for reconciled state machine

### ISSUE THREAD

**Issue by thekaveman** [openmobilityfoundation/mobility-data-specification#525]
After #506 is merged, we'll have a reconciled state machine between Agency and Provider. A follow-up task is to create additional documentation/guidance around how to translate event data from pre-1.0.0 into the newly reconciled 1.0.0 set of states.

The guidance itself should live in the [governance repo](https://github.com/openmobilityfoundation/governance) and we should link to it from the Vehicle State docs that #506 will introduce.

**Comment by schnuerle** [comment]
I will work on this this week.

**Comment by schnuerle** [comment]
I've started work on this in this wiki page. 

https://github.com/openmobilityfoundation/mobility-data-specification/wiki/Upgrading-0.4.1-to-1.0.0

**Comment by schnuerle** [comment]
I've fleshed out that page a bit more with the release.  I can update as needed but as of now it accomplishes the goals of this issue. It's linked to from the Github release page for 1.0.0.
